### PR TITLE
Enhance audio widget with allwaysreload

### PR DESCRIPTION
**Pull request by jensgulow**

### CODE REVIEW

**Comment by joltcoke** [comment]
Thanks for the pull request. Currently there are 3 issues:

1. Diff is showing most of the file a changed. Probably due to (re)formatting issues. Could you please recreate the PR with clean versions of the affected files?

2. Travis CI is failing due to formatting errors. Maybe this is related to #1.

3. Please change the parameter to "alwaysreload" (spelling fixed).

Due to Issue #1 i haven't looked at your changes.

File: src/structure/pure/Audio.js [hunk 1]
```diff
@@ -1,74 +1,82 @@
...
+    var ret_val = basicdesign.createDefaultWidget( 'audio', $e, path, flavour, type, this.update);
+    // and fill in widget specific data
+    var data = templateEngine.widgetDataInsert( path, {
+      'src'     : $e.attr('src'),
+      'id'      : $e.attr('id'),
+      'width'   : $e.attr('width'),
+      'height'  : $e.attr('height'),
+      'autoplay': $e.attr('autoplay'),
+      'loop'    : $e.attr('loop'),
+      'allwaysreload' : $e.attr('allwaysreload') ,
```
Comment (peuter) [inline src/structure/pure/Audio.js]: typo: must be alwaysreload

File: src/structure/pure/Audio.js [hunk 2]
```diff
@@ -1,74 +1,82 @@
...
+  
+  update: function(e,d) {
+    var element = $(this);
+    var actor   = element.find('.actor');
+    var value = basicdesign.defaultUpdate( e, d, element, true, element.parent().attr('id') );
+    var data  = templateEngine.widgetDataGetByElement( element );
+    var on = templateEngine.map( data[ 'threshold_value' ], data['mapping'] );
+    if (value >= on){
+      var audioWidget = document.getElementById(data['id']);
+      if (data[ 'allwaysreload' ] == 'true'){
```
Comment (peuter) [inline src/structure/pure/Audio.js]: typo: must be alwaysreload

File: src/visu_config.xsd [hunk 3]
```diff
@@ -1036,6 +1036,7 @@
     <xsd:attribute name="volume" type="xsd:string" />
     <xsd:attribute name="autoplay" type="xsd:string" />
     <xsd:attribute name="loop" type="xsd:string" />
+    <xsd:attribute name="allwaysreload" type="xsd:string" use="optional"  />
```
Comment (peuter) [inline src/visu_config.xsd]: typo: must be alwaysreload

**Comment by ChristianMayer** [comment]
@jensgulow do you intend to fix the found issues?

**Comment by ChristianMayer** [comment]
@jensgulow as we are going towards the 0.10.0 release do you intend to fix the brought up issues or do you want to close this pull request?

**Comment by peuter** [comment]
As there has been no reaction from the creator and the PR would need to be re-done for the current code base anyways I close this PR now.

**Pull request closed without merging**